src/core/core.js

```diff
@@ -429,12 +436,16 @@ define(function (require) {
     });
 
     // TODO: ???
-    if (document.readyState === 'complete') {
+    
+    if (sync) {
       this._start();
     } else {
-      window.addEventListener('load', this._start.bind(this), false);
+      if (document.readyState === 'complete') {
+        this._start();
```

Comment:
this is what's making that tests hard to write.... ```document.readyState``` is always 'complete' by the time the tests are run because gulp-mocha waits for everything to load before executing ```mocha.run()```... I tried spoofing the ```readyState``` with ```Object.defineProperty()```, but phantom raised a TypeError.  I'll try rewriting the tests with iframes tomorrow.